src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -82,8 +82,8 @@ public SqlTriggerListener(string connectionString, string tableName, string user
             }
             this._hasConfiguredMaxChangesPerWorker = configuredMaxChangesPerWorker != null;
 
-            this._scaleMonitor = new SqlTriggerScaleMonitor(this._userFunctionId, this._userTable.FullName, this._connectionString, this._maxChangesPerWorker, this._logger);
-            this._targetScaler = new SqlTriggerTargetScaler(this._userFunctionId, this._userTable.FullName, this._connectionString, this._maxChangesPerWorker, this._logger);
+            this._scaleMonitor = new SqlTriggerScaleMonitor(this._userFunctionId, this._userTable.BracketQuotedFullName, this._connectionString, this._maxChangesPerWorker, this._logger);
```

Comment:
I'd suggest passing in the entire table object - that way we're much more confident that the name is being escaped correctly (and don't accidently use the wrong name if we make changes there in the future)